packages/lesswrong/server/fmCrosspost/crosspost.ts

```diff
@@ -71,23 +68,22 @@ const updateCrosspost = async (postId: string, draft: boolean, deletedDraft: boo
 
 export const handleCrosspostUpdate = async (document: DbPost, data: Partial<DbPost>) => {
   if (
-    (data.draft !== undefined || data.deletedDraft !== undefined || data.title !== undefined) &&
+    denormalizedFieldKeys.some((key) => data[key] !== undefined) &&
     document.fmCrosspost?.foreignPostId
   ) {
-    await updateCrosspost(
-      document.fmCrosspost.foreignPostId,
-      data.draft ?? document.draft,
-      data.deletedDraft ?? document.deletedDraft,
-      data.title ?? document.title,
-    );
+    const denormalizedData = denormalizedFieldKeys.reduce(
+      (result, key) => ({...result, [key]: data[key] ?? document[key]}),
+      {},
+    ) as DenormalizedCrosspostData;
+    await updateCrosspost(document.fmCrosspost.foreignPostId, denormalizedData);
```

Comment:
I think thinking about the error I got, I think this should be in a try catch with a console.error in the catch. No need to kill kill the post update if we can't update the x-post. Maybe? I'm not sure. You could make the argument that we should, because we want to stay in sync, and for x-posts, if one site is having issues, both sites are. Ok, I've convinced myself. I take it back.

Reply:
Haha, ok! In that case, I'll leave it as it is for now. I _think_ it's probably more correct to prioritise keeping both sites in sync, but we can always make a separate PR to change this in the future if we want.